lib/ZumoOta/src/App.h

```diff
@@ -59,14 +61,14 @@ class App
 {
 public:
     /**
-     * Construct the convoy leader application.
+     * Construct the app  application.
      */
     App()
     {
     }

```

Comment:
suprised that it works with the {} here. Should be ony App(); as implemented in cpp